src/app/stores/InitialState.js

```diff
@@ -1,21 +1,17 @@
 import PropTypes from 'prop-types';
 
 export const initialSearchQuery = {
```

Comment:
I think the simplification of this part alone justifies making this change, so much clearer!